charts/orb/values.yaml

```diff
@@ -297,3 +322,48 @@ nginx_internal:
     pullPolicy: "IfNotPresent"
     repository: "nginx"
     tag: "1.21.1"
+
+keto:
+  keto:
+    autoMigrate: true
+    config:
+      serve:
+        read:
+          port: 4466
+        write:
+          port: 4467
+        metrics:
+          port: 4468
+      namespaces:
+        - id: 0
+          name: members
```

Comment:
I suppose that is missing here the field to hold the dbhost for keto, to permit us to point to external db such as RDS auth.